arrow-continuations/src/main/kotlin/arrow/continuations/generic/SuspendingComputation.kt

```diff
@@ -0,0 +1,112 @@
+package arrow.continuations.generic
+
+import kotlinx.atomicfu.atomic
+import kotlinx.atomicfu.loop
+import kotlin.coroutines.Continuation
+import kotlin.coroutines.CoroutineContext
+import kotlin.coroutines.EmptyCoroutineContext
+import kotlin.coroutines.intrinsics.COROUTINE_SUSPENDED
+import kotlin.coroutines.intrinsics.startCoroutineUninterceptedOrReturn
+import kotlin.coroutines.intrinsics.suspendCoroutineUninterceptedOrReturn
+import kotlin.coroutines.resumeWithException
+
+sealed class Either<out E, out A>
```

Comment:
I guess we can remove this sample code for Either?

Reply:
Yes, can be removed just serverd as a placeholder type.